doc/source/ray-observability/state/state-api.rst

```diff
@@ -0,0 +1,303 @@
+
+Let's list all running tasks.
+
+.. code-block:: bash
+
+    ray list tasks
+
+.. code-block:: text
+
+    ======== List: 2022-07-22 08:56:37.570942 ========
+    Stats:
+    ------------------------------
+    Total: 4
+
+    Table:
+    ------------------------------
+       FUNC_OR_CLASS_NAME        NAME                         SCHEDULING_STATE    TASK_ID                                           TYPE
+    0  task_running_300_seconds  task_running_300_seconds()   RUNNING             16310a0f0a45af5cffffffffffffffffffffffff01000000  NORMAL_TASK
+    1  task_running_300_seconds  task_running_300_seconds()   RUNNING             c8ef45ccd0112571ffffffffffffffffffffffff01000000  NORMAL_TASK
+    2  Actor.__init__            Actor.__init__()             FINISHED            ffffffffffffffff823ef6de3c08a9cb1469d36b01000000  ACTOR_CREATION_TASK
+    3  Actor.__init__            Actor.__init__()             FINISHED            ffffffffffffffff971bfac3ca40ca548209005301000000  ACTOR_CREATION_TASK
+
+You can get the state of a single task using the get API. 
+
+.. code-block:: bash
+
+    ray get tasks <TASK_ID>
+
+.. code-block:: text
+
+    ---
+    func_or_class_name: task_running_300_seconds
+    language: PYTHON
+    name: task_running_300_seconds()
+    required_resources:
+        CPU: 1.0
+    runtime_env_info:
+        runtime_env_config:
+            eager_install: true
+            setup_timeout_seconds: 600
+        serialized_runtime_env: '{}'
+    scheduling_state: RUNNING
+    task_id: 32d950ec0ccf9d2affffffffffffffffffffffff01000000
+    type: NORMAL_TASK
+
+You can also access logs through ``ray logs`` API.
+
+.. code-block:: bash
+
+    ray list actors
+    ray logs --actor-id <ACTOR_ID>
+
+.. code-block:: text
+
+    --- Log has been truncated to last 1000 lines. Use `--tail` flag to toggle. ---
+
+    :actor_name:Actor
+    Actor created
+
+
+Key Concepts
+------------
+Ray state APIs allow you to access **states** of **resources** through **summary**, **list**, and **get** APIs.
+
+- **states**: The state of the cluster of corresponding resources. States consist of immutable metadata (e.g., actor's name) and mutable states (e.g., actor's scheduling state or pid).
+- **resources**: Resources created by Ray. E.g., actors, tasks, objects, placement groups, and etc. 
+- **summary**: API to return the summarized view of resources.
+- **list**: API to return every individual entity of resources.
+- **get**: API to return a single entity of resources in detail.
+
+Summary 
+-------
+Return the summarized information of the given Ray resource (objects, actors, tasks).
+It is recommended to start monitoring states through summary APIs first. When you find anomalies
+(e.g., actors running for a long time, tasks that are not scheduled for a long time),
+you can use ``list`` or ``get`` APIs to get more details for an individual abnormal resource.
+
+E.g. Summarize all actors (e.g. number of alive actors, different actor classes, etc)
```

Comment:
is there a way we make this a header so that it appears in the content section? What do y'all think? So people get a sense of what the APIs could do w/o reading all of them. 

<img width="220" alt="image" src="https://user-images.githubusercontent.com/11676094/180499700-5ec06735-fbb7-4c2b-9e2b-80ba800d6cfd.png">

Reply:
Looks like we don't do this throughout the document. Top level headers only include new document, not headers of each doc. (and I couldn't find a way to convert it..). Will just keep it as it is..